docs/About/License.md

```diff
@@ -0,0 +1,10 @@
+# License
+
+The code for the graphical user interface portion is released under version 3.0 of the GPL.
+So every build of JackTrip with a graphical user interface is only licensed as GPL.
+With the introduction of GPL licensed dependencies in the future more parts of JackTrip
+wil become GPL. 
+
+Only files that were part of the original JackTrip project are covered by the more permissive license in LICENSE\_nogui.
+
+--8<-- "LICENSE.md"
```

Comment:
is this a valid markdown?

Reply:
You can add markdown extensions in mkdocs.yml. This one is for including files.